internal/node/running_vm.go

```diff
@@ -33,6 +33,7 @@ type runningFirecracker struct {
 	machineStarted        time.Time
 	workloadSpecification controlapi.RunRequest
 	namespace             string
+	deployedWorkload      agentapi.DeployRequest
```

Comment:
If we can consider how to consolidate the requests as mentioned above that would be fantastic. Here we refer to the `DeployRequest` as a `Workload` itself.

Reply:
Agree